src/electron/rpc/server.ts

```diff
@@ -36,27 +27,48 @@ export class RpcHandler {
 		startTime: string;
 		endTime: string;
 	}) {
+		const result = await fetchVideoMetadata(data.id);
+
+		const formats = sortBy(
+			result.streamingData.adaptiveFormats.filter((format) =>
+				format.mimeType.includes("audio/webm"),
+			),
+			(format) => -format.bitrate,
+		);
+		const format = formats[0];
+		if (!format) {
+			throw new Error("Audio data not available");
+		}
+
 		using dir = createTempDirectory();
 		const tmpFile1 = dir.join("tmp.webm");
 		const tmpFile2 = dir.join("tmp.opus");
-		const thumbnailFile = dir.join("tmp.jpg");
 		const metadataFile = dir.join("ffmetadata.txt");
 
-		// download webm audio and thumbnail via yt-dlp
-		await $("yt-dlp", [
-			data.id,
-			"--no-playlist",
-			"-f",
-			"ba[ext=webm]",
-			"-o",
+		// download webm audio
+		const audioResponse = await fetch(format.url, {
+			headers: {
+				// it looks like adding this trivial range header can make download a way faster
+				range: `bytes=0-`,
+			},
+		});
```

Comment:
`bytest=0-` seems to still get throttled if the payload is too large. Let's just port this again from https://github.com/hi-ogawa/youtube-dl-web-v2/blob/aba63b192ab7eef362aa93d0b98a6cf539284f5a/packages/app/src/utils/download.ts#L136